generator/dart/packages/generated/google_cloud_protobuf/pubspec.yaml

```diff
@@ -23,5 +23,8 @@ environment:
 
 resolution: workspace
 
+dependencies:
+  google_cloud_common: any
```

Comment:
Q: I assume `any` here means "any version".  Will we need to get more strict about these at some point?

Reply:
> IQ: I assume any here means "any version". Will we need to get more strict about these at some point?

Yes - it doesn't specify a version. We'll definitely want to use a version here in the future; I'm assuming we'd likely do something like pull the info from `generator/dart/.sidekick.toml`.